assets/app/views/project.html

```diff
@@ -4,12 +4,12 @@
       <tasks></tasks>
       <div ng-if="!renderOptions.showGetStarted" class="actions pull-right">
         <div class="btn-group">
-          <label class="btn btn-default" ng-model="$parent.overviewMode" btn-radio="'tiles'" title="Tile View">
+          <a href="" class="btn btn-default" ng-model="$parent.overviewMode" btn-radio="'tiles'" title="Tile View">
```

Comment:
Actually going to take the change to the view switcher out of this PR
because @sg00dwin is already touching those as part of his stuff

On Mon, Dec 21, 2015 at 2:19 PM, Steve Goodwin <notifications@github.com>
wrote:

> In assets/app/views/project.html
> <https://github.com/openshift/origin/pull/6417#discussion_r48179771>:
>
> > @@ -4,12 +4,12 @@
> >        <tasks></tasks>
> >        <div ng-if="!renderOptions.showGetStarted" class="actions pull-right">
> >          <div class="btn-group">
> > -          <label class="btn btn-default" ng-model="$parent.overviewMode" btn-radio="'tiles'" title="Tile View">
> > +          <a href="" class="btn btn-default" ng-model="$parent.overviewMode" btn-radio="'tiles'" title="Tile View">
>
> I think it should be like this, based on
> http://getbootstrap.com/components/#btn-groups
> Also, switching to the pf topology icon
>
> <div class="btn-group" role="group" aria-label="View Options">
>   <button type="button" class="btn btn-default">
>     <span class="fa fa-th-list" aria-hidden="true"></span>
>   </button>
>   <button type="button" class="btn btn-default">
>    <span class="pficon pficon-topology" aria-hidden="true"></span>
>   </button>
> </div>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6417/files#r48179771>.
>